test/OrleansTestKit.Tests/Grains/UnknownGrainResolver.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Orleans;
+using TestInterfaces;
+
+namespace TestGrains
+{
+    public class UnknownGrainResolver : Grain, IGrainWithStringKey
```

Comment:
Let's have a `IUnknownGrainResolver` interface that exports the `CreateAndPingMultiple` method. (I know it's not strictly necessary with the TestKit, but it's the standard Orleans pattern.)

Reply:
Done!